File: src/main/java/com/example/interviewlandbackend/dto/request/CreateContentRequest.java
package com.example.interviewlandbackend.dto.request;


import javax.validation.constraints.NotBlank;

// Request classlari ucun bos constructor ve get funksiyalarinin varligi kifayetdir
public class CreateContentRequest {

    @NotBlank
    private String contentName;


    // Bosh constructor mutleq olmalidir yoxsa body-den reqeust atmaq olmur
    public String getContentName() {
        return contentName;
    }


}

File: src/main/java/com/example/interviewlandbackend/dto/request/CreateContentRequest.kt
package com.example.interviewlandbackend.dto.request
import org.springframework.web.multipart.MultipartFile
import javax.validation.constraints.NotBlank
import javax.validation.constraints.NotNull

data class CreateContentRequest @JvmOverloads constructor(

    @field:NotBlank
    val contentName: String,
    @field:NotBlank
    val description : String,
    @field:NotNull
    val image: MultipartFile

)
